_episodes/00-intro.md

```diff
@@ -5,6 +5,7 @@ questions:
 - "How is using a cluster different than your laptop?"
 keypoints: 
 - "A cluster or distributed system is useful when your computations require more than one computer." 
+- "Because a cluster is distributed, it is only useful for certain types of computational problems."
```

Comment:
I'm okay w/ it as is.  I think we can re-formulate the key points after we actually have some lesson text so that they match.